ui/app/registration/controllers/editPatientController.js

```diff
@@ -27,8 +25,112 @@ angular.module('bahmni.registration')
             var successCallBack = function (openmrsPatient) {
                 $scope.openMRSPatient = openmrsPatient["patient"];
                 $scope.patient = openmrsPatientMapper.map(openmrsPatient);
-                console.log($scope.patient);
+                var mozAttributes = ['BI', 'NUIT', 'NUIC', 'Cedula_de_Nascimento', 'Cartao_de_Eleitor', 'Passaporte_Mocambicano'];
+                var foreignAttributes = ['DIRE', 'NUIT', 'Passaporte_Estrangeiro'];
+                $scope.editPatientDocuments = [];
+                var nationalityVar = function () {
+                    if ($scope.patient.NATIONALITY == undefined) {
+                        $scope.patient.NATIONALITY = "";
+                    }
+                    else {
+                        $scope.nationalityChoice = $scope.patient.NATIONALITY.value;
+                        if ($scope.nationalityChoice == 'Mocambicano' || $scope.nationalityChoice == 'Mozambican') {
+                            $scope.nationalityDocs = mozAttributes;
+                            $scope.existDocs = $scope.nationalityDocs;
+                        }
+                        else if ($scope.nationalityChoice == 'Estrangeiro' || $scope.nationalityChoice == 'Foreigner') {
+                            $scope.nationalityDocs = foreignAttributes;
+                            $scope.existDocs = $scope.nationalityDocs;
+                        }
+                    }
+                };
+                nationalityVar();
+
+                $scope.nationality = function () {
+                    if ($scope.patient.NATIONALITY == undefined) {
+                        $scope.patient.NATIONALITY = "";
+                    }
+                    else {
+                        $scope.nationalityChoice = $scope.patient.NATIONALITY.value;
+                        if ($scope.nationalityChoice == 'Mocambicano' || $scope.nationalityChoice == 'Mozambican') {
+                            $scope.nationalityDocs = mozAttributes;
+                            console.log($scope.nationalityDocs);
+                            $scope.existDocs = $scope.nationalityDocs;
+                        }
+                        else if ($scope.nationalityChoice == 'Estrangeiro' || $scope.nationalityChoice == 'Foreigner') {
+                            $scope.nationalityDocs = foreignAttributes;
+                            console.log($scope.nationalityDocs);
+                            $scope.existDocs = $scope.nationalityDocs;
+                        }
+                    }
+                };
+                var existingPatientDocs = function () {
```

Comment:
@amosh101 have you looked into adding unit tests for these new functions?

Reply:
It is a great idea, btw when can we schedule the call for the conventions and tips discussion?

Reply:
Yes lets schedule this for next week Monday. For now lets leave the unit tests, but I'll create some tech debt tickets to go back and implement missing tests